Sprint-2/3-mandatory-implement/3-to-pounds.js

```diff
@@ -4,3 +4,14 @@
 // You will need to declare a function called toPounds with an appropriately named parameter.
 
 // You should call this function a number of times to check it works for different inputs
+
+
+function toPound(penceString)
```

Comment:
This is functionally looking good 🙂 

I would suggest using VS Code's format document feature - this will ensure your Javascript code is following a standard format. This makes it easier for yourself, and others, to read the code in the future 🙂 

https://code.visualstudio.com/docs/editing/codebasics#_formatting